packages/svelte/src/main/ambient.d.ts

```diff
@@ -17,6 +17,23 @@ declare module '*.svelte' {
 declare function $state<T>(initial: T): T;
 declare function $state<T>(): T | undefined;
 
+declare namespace $state {
+	/**
+	 * Declares reactive state without applying reactivity to nested properties.
+	 *
+	 * Example:
+	 * ```ts
+	 * let count = $state.raw(0);
```

Comment:
we should change this example to illustrate a real use case — `0` is _already_ not proxified

(I can come back to this later if nec but I can't stick around to do it now)